Hush/Assets/Scripts/Weapon/PlayerWeaponController.cs

```diff
@@ -18,52 +18,54 @@ void Start()
     {
         equippedWeapon = sword;
         characterStats = new CharacterStats(10, 10);
-        
+
         // add any bonuses related to specific weapons
-        characterStats.AddBonus( StatType.Strength , sword.BonusDamage);
-        characterStats.AddBonus( StatType.SpellPower , spell.BonusDamage);
+        characterStats.AddBonus(StatType.Strength, sword.BonusDamage);
```

Comment:
out of curiosity; are we going to implement a project-wide conding convention? feels like we might want to (though it's super minor in this case, just in regards to some whitespace cleanup)

Reply:
I deadass am just using the Rider linter/formatter on default settings. 
 I don't think it's necessary to enforce it, but I will definitely be linting the flies I pass on